data/CMakeLists.txt

```diff
@@ -98,15 +98,13 @@ configure_file( ${CMAKE_CURRENT_SOURCE_DIR}/darktableconfig.dtd ${CMAKE_CURRENT_
 
 
 if(USE_XMLLINT)
-	execute_process(
-		COMMAND ${Xmllint_BIN} --valid ${CMAKE_CURRENT_BINARY_DIR}/darktableconfig.xml
-		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/
-		OUTPUT_QUIET ERROR_QUIET
-		RESULT_VARIABLE Xmllint_RES
-	)
-	if(NOT ${Xmllint_RES} EQUAL 0 )
-		message(FATAL_ERROR "data/darktableconfig.xml is not valid")
```

Comment:
One caveat i'm aware of - that `validate_darktableconfig_xml` target will be run **always** and not when xml is changed, that is, twice during normal build.
Is that unwanted?